src/App.tsx

```diff
@@ -1,109 +1,37 @@
 import React from 'react';
-
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import { PostList } from './components/PostList';
+import { Post } from './types/Post';
+import { User } from './types/User';
+import { Comment } from './types/Comment';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+
+function getUser(userId: number): User | null {
+  const newUser = usersFromServer.find(user => user.id === userId);
+
+  return newUser || null;
+}
+
+function getComments(id: number): Comment[] {
+  const newComment = commentsFromServer.filter(comment => (
+    comment.postId === id
+  ));
+
+  return newComment || null;
```

Comment:
```suggestion
  return newComment || null;
```
`filter` always return an Array array is a truthy value so getComments never return null

Reply:
Indeed. Got it, thanks.